devito/symbolics/printer.py

```diff
@@ -127,6 +127,9 @@ def _print_TrigonometricFunction(self, expr):
             func_name += 'f'
         return func_name + '(' + self._print(*expr.args) + ')'
 
+    def _print_Basic(self, expr):
```

Comment:
this is needed for...?

Reply:
`_print_Basic` defaults to `_print_not_supported` that appends `// Not supported in C` everywhere